docs/guides/step-by-step/step-99.md

```diff
@@ -44,7 +44,7 @@ If that feels like too much possibility to you, why not one of these options:
 -   Share your experience with Netdata and this guide. Be sure to [@mention](https://twitter.com/linuxnetdata) us on 
     Twitter!
 -   Contribute to what we do. Browse our [open issues](https://github.com/netdata/netdata/issues) and check out out
-    [contributions doc](/README.md#contribute) for ideas of how you can pitch in.
+    [contributions doc](/contribute) for ideas of how you can pitch in.
```

Comment:
```suggestion
    [contributions doc](https://learn.netdata.cloud/contribute/) for ideas of how you can pitch in.
```

`/contribute` doesn't work on GitHub, how about that^^?

Reply:
@ilyam8 Well, that is a good point. 
Do other relative file links work as long as they are in the same repo?

Reply:
> relative
> as they are in the same repo

Yes.